vehicle/raw_vehicle_cmd_converter/src/csv_loader.cpp

```diff
@@ -66,6 +66,19 @@ bool CSVLoader::validateData(const Table & table, const std::string & csv_path)
   return true;
 }
 
+Map CSVLoader::getMap(const Table & table)
```

Comment:
It seems there are so many functions which we can add const.

Reply:
they are static functions so const cannot be used.